.github/workflows/test.yml

```diff
@@ -442,7 +442,7 @@ jobs:
           channels: conda-forge,bioconda
           python-version: ${{ matrix.python-version }}
 
-      - name: Conda setup
+      - name: Set up Conda
         run: |
           conda clean -a
           conda install -n base conda-libmamba-solver
```

Comment:
this setup step can be removed if we add the `conda-solver:` to the setup-minoconda step